### DISCUSSION TITLE
Unable to render the WAV properly in FLStudio

### DISCUSSION THREAD

**Discussion by flawlesscod3r** [apohl79/audiogridder#1376] [Help]
I ran into another problem: When I try to RENDER my track, while the AG is open, first of all, it somehow adds the 40 secs of a silence, because all of the instances are suddenly starting to RELOAD (???) during the process of rendering the WAV. 

I figured out that I can make a minute of silence in the project, just to render it properly. It helped me for a while, but then, I realised that the latency is SO HUGE, that the output sound became FASTER in the moments, somewhere - OFFBEAT, somewhere - DISHARMONY or else... How to fix the LATENCY and these problems with the sound? (I'm rendering in WAV, with 512 buffer size and Dithering is ON).

In the DAW, the project sounds OK, but the render is awful :( I've read some topics from audiosex and image-line forum, and the people faced the same problem as mine, but there is no solution (I haven't found it yet, at least).

**Comment by Kramer2025** [comment]
I can second that exact behavior, especially noticeable with heavy  AA plugins.
But that's why we are using AG in first place. for the heavier stuff.

Try to experiment with fixed size buffers in FL wrapper and see if it can help temporarily.

**Reply by flawlesscod3r** [reply]
Thanks! I'll try it. Btw, running 1-2 instances of AG is OK (small 32-bit VST's). I can hear some of the sounds glitching while playing the project (an old 2015 macbook pro), but the final render sounds well). Moreover, it seems like it is unable to run some of the VST's. I hope there will be an update (haven't tried 1.6 yet).

**Reply by Kramer2025** [reply]
Check the Performance Monitor in FL to see if there are spikes. It has been very helpful to identify what's causing glitches. I doubt there will be any spikes with 32bit VSTs since they are a lot lighter on CPU but I can't be sure.


What's 1.6? I lost you there.

**Reply by flawlesscod3r** [reply]
AudioGridder v.1.6 is available. You can send a letter with a request to this email and get it: crossgridder@gmail.com

I've noticed previously, that reverbs are causing spikes.

**Reply by emack999** [reply]
Crossgridder is not audiogridder.

**Reply by Kramer2025** [reply]
I found that the hard way, when the CG dev, replied in my request mail. 😶‍🌫️
